scripts/automation/install.sh

```diff
@@ -95,8 +95,10 @@ if [ "$?" -ne "0" ]; then
     exit 7
 fi
 
-# Install the virtual environment to be the "current" if the tests pass
-(cd "${CASCADE_DEVELOP_DIR}/tests" && pytest)
+# Install the virtual environment to be the "current" if the tests pass.
+# Run testing without writing *.pyc to the __pycache__.
+PYTHONDONTWRITEBYTECODE=1
+(cd "${CASCADE_DEVELOP_DIR}/tests" && python -m pytest -p no:cacheprovider --ihme)
```

Comment:
If we're going to `chmod -R` it later, why not make cache now?

Reply:
If it doesn't pass unit tests, then this script exits before the chmod, creating a file management problem.